packages/terra-consumer-nav/src/components/nav-help/NavHelp.scss

```diff
@@ -85,7 +91,8 @@
   }
 
   .toggler-content-alignment {
-    padding: 0 40px 20px 20px;
+    margin: 0;
+    padding: 10px 40px 20px 20px;
```

Comment:
Ux really wants 10px above and 20px below? Seems strange, might want to have them look at this before this gets merged in.

Reply:
If we don't add this extra 10px it does not look good when the button is clicked in chrome. The selection border in chrome is very close to the content.Please check the  screenshot below

<img width="329" alt="screen shot 2017-12-19 at 4 12 47 pm" src="https://user-images.githubusercontent.com/21693381/34179855-8597b2b4-e4da-11e7-808f-4d443196f5fe.png">

Got +1 from Anna and Julie in the Jira.

Reply:
i was more worried about the 20 below not the 10 above

Reply:
If we consider the whole help-item including toggle button and content there is 20px both top & bottom when the toggle is open. 
<img width="323" alt="screen shot 2017-12-20 at 9 40 09 am" src="https://user-images.githubusercontent.com/21693381/34212385-23d2685e-e56a-11e7-825b-792b9945868f.png">